README.md

```diff
@@ -21,20 +21,10 @@ Before you can run the frontend app against the backend you will need to configu
 
 ### Set the Environment variables
 
-Create a copy of the .env.sample file and rename it .env and fill in below values:
+Create a copy of one of the .env.XXX files (it's best to start with the one named the same as your target environment),
```

Comment:
Hi @whi-tw - sorry if this is a silly question, but how would I know what my target environment is? I'm assuming it might typically be build (although I might be wrong about that). I'm just wondering if it might be helpful to let people know where they'd find out what their environment is.

Reply:
I had trouble with the wording for this.
I think it's supposed to mean:
> if you're running this against OIDC in `build` use `.env.build` as the template